x-pack/server/lib/create_router/index.js

```diff
@@ -9,8 +9,18 @@ import { isEsErrorFactory } from './is_es_error_factory';
 import { wrapEsError, wrapUnknownError } from './error_wrappers';
 import { licensePreRoutingFactory } from'./license_pre_routing_factory';
 
+export { wrapEsError, wrapUnknownError, wrapCustomError } from './error_wrappers';
+
+// Sometimes consumers will need to check if errors are ES errors, too.
+export const createIsEsError = server => {
```

Comment:
I think we'll want the ability to easily hand-roll our own errors when using `createRouter`, so to me it makes sense to export these helpers from the same module.

Reply:
can we just export `isEsErrorFactory` instead?
`export { isEsErrorFactory } from './is_es_error_factory';`
it is a little weird to export and import the same module in the same file though 🙂